cmd/snap/cmd_run.go

```diff
@@ -65,11 +68,12 @@ func init() {
 		func() flags.Commander {
 			return &cmdRun{}
 		}, map[string]string{
-			"command": i18n.G("Alternative command to run"),
-			"hook":    i18n.G("Hook to run"),
-			"r":       i18n.G("Use a specific snap revision when running hook"),
-			"shell":   i18n.G("Run a shell instead of the command (useful for debugging)"),
-			"strace":  i18n.G("Run the command under strace (useful for debugging"),
+			"command":    i18n.G("Alternative command to run"),
+			"hook":       i18n.G("Hook to run"),
+			"r":          i18n.G("Use a specific snap revision when running hook"),
+			"shell":      i18n.G("Run a shell instead of the command (useful for debugging)"),
+			"strace":     i18n.G("Run the command under strace (useful for debugging). Extra strace options can be specified as well here."),
```

Comment:
Nice, so this one option serves as as both the selector (don't use `strace` vs use `strace`) but also as a way to convey custom options. I think it warrants a comment for the casual reader.